compiler/modules/front.syntax/token.bal

```diff
@@ -1,6 +1,7 @@
 import wso2/nballerina.err;
 
-type Token FixedToken|readonly & VariableLengthToken;
+// JBUG #33694 can't use VariableLengthToken instead inlining
+type Token FixedToken|readonly & [IDENTIFIER, string]|readonly & [DECIMAL_NUMBER, string]|readonly & [STRING_LITERAL, string]|readonly & [HEX_INT_LITERAL, string]|readonly & [DECIMAL_FP_NUMBER, string, FpTypeSuffix?];
```

Comment:
Can we just define VariableLengthToken with each tuple readonly? (i.e. put everything after FixedToken as the definition of VariableLengthToken)

Reply:
Ahh yes, that worked, I'll change.